pyproject.toml

```diff
@@ -4,7 +4,7 @@ version = "0.8.0"
 description = "A python SDK for the Zscaler API."
 authors = ["Mitch Kelly <me@mkelly.dev>"]
 license = "MIT"
-readme = "README.md"
```

Comment:
Just noticed that I hadn't added this file to git. README.md was added to have a condensed version of GitHub's README.rst for PyPi builds. I'll add this to `develop` today. Keep this as README.md.